src/CL/Slack/Payload/AdvanceSerializeInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+
+namespace CL\Slack\Payload;
+
+
+use JMS\Serializer\Serializer;
+
+interface AdvanceSerializeInterface
```

Comment:
Can you refactor this to `AdvancedSerializerInterface` (including the extra `d` and `r`)